install_files/securedrop-ossec-server/DEBIAN/control.j2

```diff
@@ -6,7 +6,11 @@ Homepage: https://securedrop.org
 Package: securedrop-ossec-server
 Version: 3.6.0+1.6.0~rc1
 Architecture: amd64
+{% if securedrop_build_focal_support %}
+Depends: libevent-2.1.7,libpcre2-8-0,ossec-server,securedrop-keyring,securedrop-config
+{% else %}
 Depends: libevent-1.4-2,libpcre2-8-0,ossec-server,securedrop-keyring,securedrop-config
```

Comment:
As above, perhaps we can standardize on a single `libevent` metapackage that would work well. Can you check whether that's possible and add if so?

Reply:
```
# apt install libevent    
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libevent
```

This is also not possible due to package naming.